mkosi/resources/mkosi.md

```diff
@@ -1170,43 +1170,43 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
-  | `openssh`           | X      | X      | X      | X      | X        |
-  | `openssl`           | X      | X      | X      | X      | X        |
-  | `pacman`            | X      |        | X      | X      |          |
-  | `pesign`            |        | X      |        |        |          |
-  | `qemu`              | X      | X      | X      | X      | X        |
-  | `sbsigntools`       | X      |        | X      | X      | X        |
-  | `socat`             | X      | X      | X      | X      | X        |
-  | `squashfs-tools`    | X      | X      | X      | X      | X        |
-  | `strace`            | X      | X      | X      | X      | X        |
-  | `swtpm`             | X      | X      | X      | X      | X        |
-  | `systemd`           | X      | X      | X      | X      | X        |
-  | `ukify`             | X      |        | X      | X      | X        |
-  | `tar`               | X      | X      | X      | X      | X        |
-  | `util-linux`        | X      | X      | X      | X      | X        |
-  | `virtiofsd`         | X      | X      |        | X      | X        |
-  | `xfsprogs`          | X      | X      | X      | X      | X        |
-  | `xz`                | X      | X      | X      | X      | X        |
-  | `zstd`              | X      | X      | X      | X      | X        |
-  | `zypper`            | X      |        | X      | X      |          |
+  |                     | Fedora | CentOS | Debian | Arch | openSUSE |
+  |---------------------|--------|--------|--------|------|----------|
+  | `apt`               | X      | X      | X      | x    |          |
+  | `archlinux-keyring` | X      |        | X      | X    |          |
+  | `bash`              | X      | X      | X      | X    | X        |
+  | `btrfs-progs`       | X      |        | X      | X    | X        |
+  | `bubblewrap`        | X      | X      | X      | X    | X        |
+  | `ca-certificates`   | X      | X      | X      | X    | X        |
+  | `coreutils`         | X      | X      | X      | X    | X        |
+  | `cpio`              | X      | X      | X      | X    | X        |
+  | `curl`              | X      | X      | X      | X    | X        |
+  | `debian-keyring`    | X      | X      | X      | X    |          |
+  | `dnf`               | X      | X      | X      | X    | X        |
+  | `dosfstools`        | X      | X      | X      | X    | X        |
+  | `e2fsprogs`         | X      | X      | X      | X    | X        |
+  | `edk2-ovmf`         | X      | X      | X      | X    | X        |
+  | `erofs-utils`       | X      |        | X      | X    | X        |
+  | `mtools`            | X      | X      | X      | X    | X        |
+  | `openssh`           | X      | X      | X      | X    | X        |
+  | `openssl`           | X      | X      | X      | X    | X        |
+  | `pacman`            | X      |        | X      | X    |          |
+  | `pesign`            | x      | X      | x      | x    | x        |
```

Comment:
Lower case 'x'?

Reply:
Guess I will need to increase my font size...